src/config-hardware/printers.md

```diff
@@ -0,0 +1,124 @@
+# Setting up Printers
+
+Most printers are plug-and-play in Pop!\_OS. Add a network or USB connected printers in `Settings` ➞ `Printers`. Clicking `Add a Printer` will automatically search for any connected printers. Printers can also be added manually by clicking `Additional Printer Settings`.
```

Comment:
> Add a network or USB connected printers

This clause contains a plurality conflict.

> will automatically search for any connected printers.

Was the first word supposed to be `adding` instead of `add`?

Reply:
> This clause contains a plurality conflict.

Yea this is a typo. Sometimes I add an 's' at the end when I'm typing too fast. I'll try to watch for this since it happened a few times in this section. Thanks for catching these.

> Was the first word supposed to be adding instead of add?

I was going with the imperative for consistency. I've been starting all section intros this way. If it seems awkward I can create an issue to rewrite these.